### DISCUSSION TITLE
How can I export an outline without bullet points?

### DISCUSSION THREAD

**Discussion by cristianconti** [vslinko/obsidian-outliner#177] [Q&A] [answered]
I'm a newbie in Obsidian and markdown world, so please forgive me if it's a dumb question.
I found that outlining ideas and text are my way to writing, rearranging text, and so on.

Question: How can i export outlines without all the bullets?

**Comment by vslinko** [comment, marked as answer]
@cristianconti Hello!

Sorry, but I don't know the right answer. When I need it, I convert my outlines to regular texts manually, by removing bullets or by rewriting completely.
